### PR TITLE
chore(frontend): improve camp dashboard performance

### PR DESCRIPTION
**Pull request by usu**
Had to base this on #2683, look for [last commit](https://github.com/ecamp/ecamp3/pull/2763/commits/5fe2f3a0566d47aaa591d1dc440f442831e633a8) only

Improves performance of dashboard view. Limits to 3 network calls instead if individual calls to each activity and each scheduleEntry.

A potential improvement could be to only load activities from API which I'm responsible for. However, impact is not large. And as an additional benefit, all data is already loaded if I switch to the picasso afterwards.

`day.scheduleEntries()` is now calcuated in memory, similar as in print:
https://github.com/ecamp/ecamp3/blob/devel/print/components/ProgramDay.vue#L36

This relation might also benefit, if we return individual scheduleEntry links instead of 1 single collection link (similar as contentNode->children). 
@carlobeltrame: What do you think on this?

### CODE REVIEW

File: frontend/src/views/camp/Dashboard.vue [hunk 1]
```diff
@@ -27,7 +29,7 @@ Admin screen of a camp: Displays details & periods of a single camp and allows t
                     <v-col :key="day._meta.self" cols="12" class="pb-0">
                       <h4>{{ day.dayOffset + 1 }}) {{ dateLong(day.start) }}</h4>
                     </v-col>
-                    <template v-for="scheduleEntry in day.scheduleEntries().items">
+                    <template v-for="scheduleEntry in scheduleEntriesByDay(day)">
```
Comment (manuelmeister) [inline frontend/src/views/camp/Dashboard.vue]: Can this be cached? I thought this would loop, as it is not a computed property.

**Review by manuelmeister — CHANGES_REQUESTED** [review]

**Review by BacLuc — APPROVED** [review]

File: frontend/src/views/camp/Dashboard.vue (hunk 1 above)
Comment (usu) [reply frontend/src/views/camp/Dashboard.vue]: I don't think this method is very expensive, but didn't do a deeper analysis. Any good proposal on how to cache?

Comment (manuelmeister) [reply frontend/src/views/camp/Dashboard.vue]: Wouldn't this be better with a computed property? Or am I missing something?

**Review by manuelmeister — APPROVED** [review]
Nice
Minor: use reduce

**Review by pmattmann — APPROVED** [review]

**Pull request merged**